examples/tof-viewer/src/ADIMainWindow.cpp

```diff
@@ -1710,11 +1710,15 @@ void ADIMainWindow::displayActiveBrightnessWindow(
         CaptureABVideo();
 
         bool logImage = view->getLogImage();
+        bool autoScale = view->getAutoScale();
         ImGui::Checkbox("Log Image", &logImage);
```

Comment:
wouldn't make more sense to hide the log image checkbox if the autoScale one isn't enabled? It might be confusing for the users if they press it and nothing happens.

Reply:
Done. I hide log image checkbox if autoscale checkbox is disabled.